src/components/com_tjcertificate/site/views/certificate/view.html.php

```diff
@@ -93,7 +95,8 @@ public function display($tpl = null)
 
 		$certificateUrl = 'index.php?option=com_tjcertificate&view=certificate&certificate=' . $this->certificate->unique_certificate_id;
 		$this->certificateUrl = Uri::root() . substr(Route::_($certificateUrl), strlen(Uri::base(true)) + 1);
-		$this->downloadPermission = $certificate::canDownload($this->certificate->unique_certificate_id);
+		$this->downloadPermission = $this->certificate->canDownload();
+		$this->linkedInProfileUrl = $this->certificate->getLinkedInProfileUrl();
```

Comment:
I think this should be conditional based on the global config

Reply:
done